cmd/promoted-image-governor/main.go

```diff
@@ -355,6 +350,11 @@ func main() {
 		return
 	}
 
+	kubeConfig, err := clientcmd.BuildConfigFromFlags("", opts.kubeconfig)
```

Comment:
Why is this needed? It always has a kubeconfig or not?

Reply:
It loads from inCluster config if we do not provide it from flag.